### ISSUE TITLE
Safari performance is poor

### ISSUE THREAD

**Issue by kylemcdonald** [justadudewhohacks/face-api.js#15]
There is a known bug with tensorflow.js on Safari that keeps it from performing accurately (on mobile and Desktop). The good news is it was just recently fixed https://github.com/tensorflow/tfjs-core/pull/978 The changes are merged into master but not released as a new version yet.

Here's what the difference looks like in practice. Chrome:

<img width="575" alt="screen shot 2018-06-21 at 17 00 33" src="https://user-images.githubusercontent.com/157106/41745516-2bba6652-7575-11e8-89ad-5a1adf5a30bf.png">

Safari:

<img width="582" alt="screen shot 2018-06-21 at 17 00 27" src="https://user-images.githubusercontent.com/157106/41745524-3272a522-7575-11e8-9917-bf627155a79e.png">

**Comment by kylemcdonald** [comment]
I changed this dependency: "@tensorflow/tfjs-core": "0.11.9"

And now it works on Desktop Safari, the results are almost exactly the same as Chrome:

<img width="590" alt="screen shot 2018-06-22 at 12 03 34" src="https://user-images.githubusercontent.com/157106/41786870-abeb50d0-7614-11e8-923f-f6cdae42d3d7.png">

Results on iOS Chrome and Safari are the same as well.

**Comment by justadudewhohacks** [comment]
Hmm okay, is that actually a tfjs issue and it's simply a matter of updating the tfjs-core version once they released a fix?

**Comment by kylemcdonald** [comment]
yes, tfjs-core is now updated (version 0.11.9 has the fix for safari).

however it looks like there's still a bug inside face-api.js that's keeping it from running correctly on iOS (that's the trace i pasted above).

**Comment by vbh25** [comment]
Have you guys found any solution to make it work on ios mobile ?

**Comment by justadudewhohacks** [comment]
I didn't, but if anyone has access to an IOS device + safari and can figure out, which ops cause issues on safari, we could report an issue at tfjs-core.

**Comment by irony** [comment]
Do you have any plans to merge to new tfjs wasm backend? The facemesh seems to be a good place to start?

https://blog.tensorflow.org/2020/03/face-and-hand-tracking-in-browser-with-mediapipe-and-tensorflowjs.html
